synapse/storage/data_stores/main/schema/full_schemas/README.md

```diff
@@ -1,13 +1,23 @@
 # Building full schema dumps
 
-These schemas need to be made from a database that has had all background updates run.
+These schemas need to be made from a database that has had all background
+updates run. The schemas are used at ...
```

Comment:
at ... ?

Reply:
the question is, why is this schema here in the repo? Is it used somewhere? Or just for information if someone is looking into the database structure?

Reply:
how do you think synapse sets up its database?